factory/__main__.py

```diff
@@ -1,37 +1,79 @@
-import os
 import signal
+import argparse
 import sys
+import os
 from pathlib import Path
 
 from ImageGoNord import GoNord
 
 from rich.console import Console
 from rich.panel import Panel
-from rich.prompt import IntPrompt
-from rich.progress import track
+
 
 def main():
-    signal.signal(signal.SIGINT, signal_handler)
-    console = Console()
 
-    global gruvbox_factory
+    global gruvbox_factory, console
     gruvbox_factory = GoNord()
     gruvbox_factory.reset_palette()
     add_gruvbox_palette()
 
+    signal.signal(signal.SIGINT, signal_handler)
+    console = Console()
+
+    # Checks if there's an argument
+    if len(sys.argv) > 1:
+        image_file = fromCommandArgument()
+    else:
+        image_file = fromTui()
+    process_image(image_file)
+
+
+
+# Gets the file path from the Argument
+def fromCommandArgument():
+
+    command_parser = argparse.ArgumentParser(
+        description='A simple cli to convert manufacture a gruvbox themed wallpaper;'
```

Comment:
here it should be "A simple cli to manufacture a gruvbox themed wallpaper", without the **convert**